<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="playXButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>154</width>
     <height>55</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>30</horstretch>
     <verstretch>100</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>154</width>
     <height>55</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
 border-top:0px;
 font-size:20px;
 text-align:center;
   font-family:Comic Sans MS;
   border: 2px solid black;
   border-radius:20%;
   min-width:150;
   max-width:150;
   min-height:51;
   max-height:51;
}

QPushButton:hover {
	background-color:black;
    color:white;
    widget-animation-duration: 1000;

}</string>
   </property>
   <property name="text">
    <string>Play X</string>
   </property>
  </widget>
  <widget class="QPushButton" name="playOButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>170</y>
     <width>154</width>
     <height>55</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
 border-top:0px;
 font-size:20px;
 text-align:center;
   font-family:Comic Sans MS;
   border: 2px solid black;
   border-radius:20%;
   min-width:150;
   max-width:150;
   min-height:51;
   max-height:51;
}
QPushButton:hover {
	background-color:black;
    color:white;
    widget-animation-duration: 1000;

}</string>
   </property>
   <property name="text">
    <string>Play O</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
font-size:30px;
font-family: Comic Sans MS;
}</string>
   </property>
   <property name="text">
    <string>Tic Tac Toe</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>283</width>
     <height>75</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">QLabel {
font-size:20px;
font-family: Comic Sans MS;
}</string>
      </property>
      <property name="text">
       <string>AI Difficulty</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>5</number>
      </property>
      <item>
       <widget class="QPushButton" name="easyButton">
        <property name="styleSheet">
         <string notr="true">QPushButton {
	   background-color: lightgreen;
	   border-radius: 10px;
      padding: 5px;
     border: 2px solid green;
     background: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 1, stop: 0 lightgreen, stop: 1 green);
    font-size:13px;
    font-family: Comic Sans MS;
    font-weight:bold;
}


QPushButton:hover {
    background-color:darkgreen;
    border: 1px solid black;
}

</string>
        </property>
        <property name="text">
         <string>Easy</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="mediumButton">
        <property name="styleSheet">
         <string notr="true">QPushButton {
	   background-color: lightyellow;
	   border-radius: 10px;
      padding: 5px;
     border: 2px solid orange;
     background:  qlineargradient(x1: 0, y1: 0, x2: 1, y2: 1, stop: 0 lightyellow, stop: 1 orange);
    font-size:13px;
    font-family: Comic Sans MS;
    font-weight: bold;
}


QPushButton:hover {
    background-color: darkorange; 
    border: 1px solid black;
}

</string>
        </property>
        <property name="text">
         <string>Medium</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="hardButton">
        <property name="styleSheet">
         <string notr="true">QPushButton {
	   background-color: lightcoral;
	   border-radius: 10px;
      padding: 5px;
     border: 2px solid red;
     background: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 1, stop: 0 lightcoral, stop: 1 red);
    font-size:13px;
    font-family: Comic Sans MS;
    font-weight: bold;
}


QPushButton:hover {
    background-color: darkred;
    border: 1px solid black;
}

</string>
        </property>
        <property name="text">
         <string>Hard</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
